js/nova.js

```diff
@@ -273,6 +275,75 @@ module.exports = class nova extends Exchange {
         };
     }
 
+    parseTransaction (transaction, currency = undefined) {
+        let timestamp = this.safeInteger2 (transaction, 'unix_t_time_seen');
+        if (timestamp !== undefined) {
+            timestamp *= 1000;
+        }
+        let updated = this.safeInteger (transaction, 'unix_t_time_sent');
+        if (updated !== undefined) {
+            updated *= 1000;
+        }
+        const currencyId = this.safeString (transaction, 'currency');
+        const code = this.safeCurrencyCode (currencyId);
+        const status = this.parseTransactionStatus (this.safeString (transaction, 'status'));
+        const amount = this.safeFloat (transaction, 'tx_amount');
+        const address = this.safeString (transaction, 'tx_address');
```

Comment:
Hey @cod3gen ! Thx for the PR, as always! ) A quick question: can you plz paste sample structures for deposits and withdrawals here? We need to know if the address is the receiver's address or the sender's address, among other things. Thx again!

Reply:
Hi @kroitor! NP, Happy to help :-)

Deposit response;

> [31]=>
>   array(13) {
>     ["info"]=>
>     array(9) {
>       ["currency"]=>
>       string(4) "LINX"
>       ["currencyid"]=>
>       int(1123)
>       ["status"]=>
>       string(9) "Accounted"
>       ["time_seen"]=>
>       string(19) "2019-07-15 02:35:17"
>       ["tx_address"]=>
>       string(34) "XGczmz4pZie31biCUpJJFAcohpSgAUkUzx"
>       ["tx_amount"]=>
>       string(11) "25.09363200"
>       ["tx_txid"]=>
>       string(64) "bfd4fbb0938452641e243d8c4696313c6fcd87d92296638f3072c154c07e62d9"
>       ["unix_t_time_seen"]=>
>       int(1563150917)
>       ["type"]=>
>       string(7) "deposit"
>     }
>     ["id"]=>
>     NULL
>     ["currency"]=>
>     string(4) "LINX"
>     ["amount"]=>
>     float(25.093632)
>     ["address"]=>
>     string(34) "XGczmz4pZie31biCUpJJFAcohpSgAUkUzx"
>     ["tag"]=>
>     NULL
>     ["status"]=>
>     string(2) "ok"
>     ["type"]=>
>     string(7) "deposit"
>     ["updated"]=>
>     NULL
>     ["txid"]=>
>     string(64) "bfd4fbb0938452641e243d8c4696313c6fcd87d92296638f3072c154c07e62d9"
>     ["timestamp"]=>
>     int(1563150917000)
>     ["datetime"]=>
>     string(24) "2019-07-15T00:35:17.000Z"
>     ["fee"]=>
>     array(2) {
>       ["currency"]=>
>       string(4) "LINX"
>       ["cost"]=>
>       NULL
>     }
>   }
> 

<< tx_address >> is receiver's address for deposits.

Withdrawals:
array(1) {
  [0]=>
  array(13) {
    ["info"]=>
    array(11) {
      ["currency"]=>
      string(3) "ABY"
      ["currencyid"]=>
      int(364)
      ["status"]=>
      string(8) "Approved"
      ["time_requested"]=>
      string(19) "2019-07-28 11:23:10"
      ["time_sent"]=>
      string(4) "None"
      ["tx_address"]=>
      string(34) "AGrAwSgnQKMC8h5GJCpWKsaNxvinxBk2wY"
      ["tx_amount"]=>
      string(13) "1003.21921401"
      ["tx_txid"]=>
      NULL
      ["unix_t_daterequested"]=>
      int(1564305790)
      ["unix_t_datesent"]=>
      NULL
      ["type"]=>
      string(10) "withdrawal"
    }
    ["id"]=>
    NULL
    ["currency"]=>
    string(3) "ABY"
    ["amount"]=>
    float(1003.21921401)
    ["address"]=>
    string(34) "AGrAwSgnQKMC8h5GJCpWKsaNxvinxBk2wY"
    ["tag"]=>
    NULL
    ["status"]=>
    string(8) "Approved"
    ["type"]=>
    string(10) "withdrawal"
    ["updated"]=>
    NULL
    ["txid"]=>
    NULL
    ["timestamp"]=>
    NULL
    ["datetime"]=>
    NULL
    ["fee"]=>
    array(2) {
      ["currency"]=>
      string(3) "ABY"
      ["cost"]=>
      NULL
    }
  }
}

Withdrawals is also showing receiver address. Pending a withdrawal now to see what "status" field will be when they have processed it. Fixed timestamps and added << Approved >> to parseTransactionStatus also.

Im also adding fetchOpenOrders function, shall i add this to this pull request or open a new one?

Reply:
> shall i add this to this pull request or open a new one?

I could merge them a bit faster, if we keep concerns separated. So, if you can add one more PR  for the fetchOpenOrders – that would be fantastic! )